### PR TITLE
coqPackages: doc

### PR DESCRIPTION
**Pull request by CohenCyril**
###### Motivation for this change

Changing bullet style to fix the rendering
cf https://github.com/NixOS/nixpkgs/pull/105877#issuecomment-757434759

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
You can test to see if this worked by running this in your checkout of Nixpkgs.

```ShellSession
$ cd doc
$ nix-build && nix-shell -p python3 --run 'python3 -m http.server --directory ./result/share/doc/nixpkgs'
```

For me it still renders incorrectly.

Screen Shot 2021-01-10 at 19 47 26

**Comment by siraben** [comment]
You may want to change the type of list using the admonitions added by @jtojnar used for instance in

```
`system`

: This is a two-component shorthand for the platform. Examples of this would be "x86_64-darwin" and "i686-linux"; see `lib.systems.doubles` for more. The first component corresponds to the CPU architecture of the platform and the second to the operating system of the platform (`[cpu]-[os]`). This format has built-in support in Nix, such as the `builtins.currentSystem` impure string.

`config`

: This is a 3- or 4- component shorthand for the platform. Examples of this would be `x86_64-unknown-linux-gnu` and `aarch64-apple-darwin14`. This is a standard format called the "LLVM target triple", as they are pioneered by LLVM. In the 4-part form, this corresponds to `[cpu]-[vendor]-[os]-[abi]`. This format is strictly more informative than the "Nix host double", as the previous format could analogously be termed. This needs a better name than `config`!

`parsed`

: This is a Nix representation of a parsed LLVM target triple with white-listed components. This can be specified directly, or actually parsed from the `config`. See `lib.systems.parse` for the exact representation.
```

This renders as
Screen Shot 2021-01-10 at 19 49 44

@ryantm do you think this is appropriate?

File: doc/languages-frameworks/coq.section.md [hunk 1]
```diff
@@ -3,37 +3,37 @@
 ## Coq derivation: `coq`
 
 The Coq derivation is overridable through the `coq.override overrides`, where overrides is an attribute set which contains the arguments to override. We recommend overriding either of the following
-+ `version` (optional, defaults to the latest version of Coq selected for nixpkgs, see `pkgs/top-level/coq-packages` to witness this choice), which follows the conventions explained in the `coqPackages` section below,
```
Comment (jtojnar) [inline doc/languages-frameworks/coq.section.md:6]: You would likely need an empty line between the paragraph and the list item. This is why I prefer Docbook over Markdown :laughing:

**Comment by CohenCyril** [comment]
I don't understand, despite this PR, https://nixos.org/manual/nixpkgs/unstable/#sec-language-coq is still broken...

**Comment by jtojnar** [comment]
Perhaps it did not update yet? You can test it locally by running `nix-build pkgs/top-level/release.nix -A manual` in the nixpkgs directory.

**Pull request merged**